test/query.mjs

```diff
@@ -321,9 +322,44 @@ describe("query", function () {
     });
 
     // Add FormatQueryInfo to container in order to test which options are were passed
-    it.skip("can set the include option to an array");
-    it.skip("can specify '--include all' to set all include options");
-    it.skip("can specify '--include none' to set no include options");
+    it("can set the include option to '[summary]' by default", async function () {
+      const formatQueryInfo = container.resolve("formatQueryInfo");
+
+      await run(`query "foo" --secret=foo`, container);
+
+      expect(formatQueryInfo.getCall(0).args[1].include).to.deep.equal([
+        "summary",
+      ]);
+    });
```

Comment:
these tests are a good first start (and fine for now - no reason to re-do this PR), but in general, i think it gives better confidence to test a little deeper. instead of "did my work function receive the right args," you can mock the network call and assert that stderr received the right output - which confirms that _after_ receiving the args, your work function actually did the right thing.

Reply:
We do a little bit of that already, just not with different permutations of args.

https://github.com/fauna/fauna-shell/blob/feca5764e2f38ca34722ec3279a5056db36943e4/test/query.mjs#L328-L383

I'll add one more. Easy peasy.